src/components/FindWallet/WalletFilterSidebar.tsx

```diff
@@ -0,0 +1,496 @@
+          filterKey: "gas_fee_customization",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "ENS support",
+          icon: <ENSSupport />,
+          description:
+            "The wallet supports sending transactions to ENS addresses.",
+          filterKey: "ens_support",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "Token importing",
+          icon: <ERC20Support />,
+          description:
+            "Can import ERC-20 token contract addresses into the wallet.",
+          filterKey: "erc_20_support",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+    {
+      title: "Onboard / offboard",
+      open: true,
+      items: [
+        {
+          title: "Buy crypto",
+          icon: <BuyCrypto />,
+          description:
+            "User is able to buy crypto with fiat directly in the wallet.",
+          filterKey: "buy_crypto",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "Withdraw crypto",
+          icon: <WithdrawCrypto />,
+          description:
+            "User is able to withdraw to fiat directly in the wallet.",
+          filterKey: "withdraw_crypto",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+    {
+      title: "Smart contracts",
+      open: true,
+      items: [
+        {
+          title: "Multisig",
+          icon: <Multisig />,
+          description:
+            "Wallets that require two or more signatures from private keys for a transaction.",
+          filterKey: "multisig",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "Social recovery",
+          icon: <SocialRecover />,
+          description:
+            "Wallets that allow guardians to change the signing key for smart contract wallets.",
+          filterKey: "social_recovery",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+  ])
+
+  const setOpen = (idx) => {
+    const updatedFilterOptions = [...filterOptions]
+    updatedFilterOptions[idx].open = !updatedFilterOptions[idx].open
+    setFilterOptions(updatedFilterOptions)
+  }
+
+  const setShowOptions = (idx, itemidx, value) => {
+    const updatedFilterOptions = [...filterOptions]
+    updatedFilterOptions[idx].items[itemidx].showOptions =
+      !updatedFilterOptions[idx].items[itemidx].showOptions
+    setFilterOptions(updatedFilterOptions)
+
+    const keys = updatedFilterOptions[idx].items[itemidx].options.map(
+      (item) => item.filterKey
+    )
+    updateFilterOptions(keys, value)
+  }
+
+  return (
+    <Container>
+      {filterOptions.map((filterOption, idx) => {
+        return (
+          <FilterPanel>
+            <Header
+              onClick={() => {
+                setOpen(idx)
+              }}
+            >
```

Comment:
Lets SRs know this item is collapsible/expandable

```suggestion
            <Header
              onClick={() => {
                setOpen(idx)
              }}
              role="button"
              aria-expanded={filterOption.open ? "true" : "false"}
            >
```